### ISSUE TITLE
[A11Y] Missing skip links

### ISSUE THREAD

**Issue by erikkroes** [OpenConext/OpenConext-myconext#1059]
## Finding
The page does not have a skip link

## Location
All

## Problem
A skip link allows users that navigate with a keyboard to skip navigation and other repeated noise.

## Solution
Check all locations and add a skip link when needed
- eduID.nl (public gui)
- login.eduid.nl (account gui)
- mijn.eduid.nl (myconext gui)

## Resources
- https://www.solidstart.info/#keyboard
- https://www.erikkroes.nl/glossary/#skip-link
- https://www.w3.org/WAI/WCAG22/Understanding/bypass-blocks.html 
- https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html
